lib/vector-core/src/event/metadata.rs

```diff
@@ -1,19 +1,19 @@
 #![deny(missing_docs)]
 
-use std::{borrow::Cow, collections::BTreeMap, fmt, sync::Arc};
+use super::{BatchNotifier, EventFinalizer, EventFinalizers, EventStatus, ObjectMap};
+use crate::{
+    config::{LogNamespace, OutputId},
+    schema,
+};
```

Comment:
Ditto this reordering. Simply leaving a blank line between the groups would have resolved it.